addons/assignGear/functions/fnc_assignGearVehicle.sqf

```diff
@@ -91,35 +95,270 @@ private _transportItems = getArray(_path >> "TransportItems");
-    nil
-} count _transportMagazines; // count used here for speed, make sure nil is above this line
-
-// transportItems
-{
-    (_x splitString ":") params ["_classname", ["_amount", "1", [""]]];
-    _theVehicle addItemCargoGlobal [_classname, parseNumber _amount];
-    nil
-} count _transportItems; // count used here for speed, make sure nil is above this line
-
-// transportWeapons
-{
-    (_x splitString ":") params ["_classname", ["_amount", "1", [""]]];
-    private _disposableName = [cba_disposable_LoadedLaunchers, _classname, "get", ""] call FUNC(getDisposableInfo);
-    if (_disposableName != "") then {
-        TRACE_2("cba_disposable_LoadedLaunchers replace",_classname,_disposableName);
-        _classname = _disposableName;
+switch (GVAR(setVehicleLoadouts)) do {
+    case 1: { // ammo in vehicle inventory
+        // transportMagazines
+        {
+            (_x splitString ":") params ["_classname", ["_amount", "1", [""]]];
+            _theVehicle addMagazineCargoGlobal [_classname, parseNumber _amount];
+            nil
+        } count _transportMagazines; // count used here for speed, make sure nil is above this line
+
+        // transportItems
+        {
+            (_x splitString ":") params ["_classname", ["_amount", "1", [""]]];
+            _theVehicle addItemCargoGlobal [_classname, parseNumber _amount];
+            nil
+        } count _transportItems; // count used here for speed, make sure nil is above this line
+
+        // transportWeapons
+        {
+            (_x splitString ":") params ["_classname", ["_amount", "1", [""]]];
+            private _disposableName = [cba_disposable_LoadedLaunchers, _classname, "get", ""] call FUNC(getDisposableInfo);
+            if (_disposableName != "") then {
+                TRACE_2("cba_disposable_LoadedLaunchers replace",_classname,_disposableName);
+                _classname = _disposableName;
+            };
+            _theVehicle addWeaponCargoGlobal [_classname, parseNumber _amount];
+            nil
+        } count _transportWeapons; // count used here for speed, make sure nil is above this line
+
+        // transportBackpacks
+        {
+            (_x splitString ":") params ["_classname", ["_amount", "1", [""]]];
+            _theVehicle addBackpackCargoGlobal [_classname, parseNumber _amount];
+            nil
+        } count _transportBackpacks; // count used here for speed, make sure nil is above this line
     };
-    _theVehicle addWeaponCargoGlobal [_classname, parseNumber _amount];
-    nil
-} count _transportWeapons; // count used here for speed, make sure nil is above this line
-
-// transportBackpacks
-{
-    (_x splitString ":") params ["_classname", ["_amount", "1", [""]]];
-    _theVehicle addBackpackCargoGlobal [_classname, parseNumber _amount];
-    nil
-} count _transportBackpacks; // count used here for speed, make sure nil is above this line
+    case 2: { // ammo in boxes in vehicle
+        private _boxClassname = "Box_NATO_Ammo_F";
```

Comment:
Is this meant to be a public variable, or just a placeholder while you were working on it? It looks like you use it once but it might be kinda cool to be able to change the box type.

Reply:
it was meant to be a semi-constant variable. i didn't want to figure out the UX of deciding which ammo box to spawn, so I left it as a variable so it would be 1) easily changed into a GVAR later 2) clearly defined in code so in a later PR someone could change it rapidly

Reply:
ended up adding it anyway